fish_prompt.fish

```diff
@@ -12,7 +12,7 @@ set -g __pure_fresh_session 1
 
 __pure_set_default pure_symbol_prompt "❯"
 __pure_set_default pure_symbol_git_down_arrow "⇣"
-__pure_set_default pure_symbol_git_down_arrow "⇡"
+__pure_set_default pure_symbol_git_up_arrow "⇡"
```

Comment:
I think this is the right line, because this is the only change that doesn't exactly fit with the rest of the squashed commits.

If you merge the change in this line along with the rest of the `string` changes, fish 2.2 users will only be able to use the current commit (dfe81b9) or older. Unfortunately for them, dfe81b9 has this bug in it.

By taking only this line out of this squash and merging it before the rest of the changes, the last commit that works for fish 2.2 users will have correct code.
Then you can everything else and 2.3+ users will get both correct code and better performance.